### PR TITLE
supervisor-release: drop entire resource & references

### PR DESCRIPTION
**Pull request by xginn8**
we will move away from a dedicated resource in favor of native releases

HQ: https://github.com/balena-io/balena-io/pull/2177
Change-type: minor
Signed-off-by: Matthew McGinn <matthew@balena.io>

### CODE REVIEW

File: src/migrations/00031-drop-supervisor-release.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+DROP TABLE "supervisor release";
```
Comment (Page-) [inline src/migrations/00031-drop-supervisor-release.sql:1]: This needs to be the last bit, after all the things that depend on it have been dropped

Comment (xginn8) [reply src/migrations/00031-drop-supervisor-release.sql]: updated.

**Comment by xginn8** [comment]
this PR is pending an arch call

**Comment by Page-** [comment]
@balena-ci republish

**Comment by thgreasi** [comment]
@xginn8 was this good to be merged?
The UI still has references to that resource & the respective fields of the device resource.
See: https://github.com/balena-io/balena-ui/search?q=should_be_managed_by__supervisor_release+OR+supervisor_release&unscoped_q=should_be_managed_by__supervisor_release+OR+supervisor_release

**Comment by xginn8** [comment]
@thgreasi thanks for checking -- we are moving the resource back to the closed api for the time being, see https://github.com/balena-io/balena-api/pull/2662.

**Comment by thgreasi** [comment]
Oh I missed that. Thanks for clarifying :+1:

**Pull request merged**